block/submit.go

```diff
@@ -297,3 +321,31 @@ func (m *Manager) UpdateLastSubmittedHeight(event pubsub.Message) {
 		}
 	}
 }
+
+// SetLastSettlementBlockTime saves the last block on SL timestamp
+func (m *Manager) SetLastSettlementBlockTime(time time.Time) error {
+	_, err := m.Store.SaveLastSettlementBlockTime(uint64(time.UTC().UnixNano()), nil)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// GetLastSettlementBlockTime returns the last block on  SL timestamp
+func (m *Manager) GetLastSettlementBlockTime() (time.Time, error) {
+	lastSettlementBlockTime, err := m.Store.LoadLastSettlementBlockTime()
```

Comment:
I guess this could error in case no sync happened yet from settlement or we didn't submit (i.e sequencer restarted so already synced).
in that case it may fail the submission loop.
I think we should either way be more tolerant torwards error here and simply log them and not fail critical flows.

Reply:
its been refactored so it is not needed to store anything.